examples/custom_shape/double_wave.py

```diff
@@ -0,0 +1,49 @@
+import math
+from locust import HttpUser, TaskSet, task, constant
+from locust import LoadTestShape
+
+
+class UserTasks(TaskSet):
+    @task
+    def get_root(self):
+        self.client.get("/")
+
+
+class WebsiteUser(HttpUser):
+    wait_time = constant(0.5)
+    tasks = [UserTasks]
+
+
+class DoubleWave(LoadTestShape):
+    """
+    A shape to immitate some specific user behaviour. In this example, midday
+    and evening meal times.
+
+    Keyword arguments:
+
+        min_users -- minimum users
+        peak_one -- first peak size
```

Comment:
peak_one sounds to me like "the number of users at the first peak"

Not sure what would be a better name though.

Reply:
I just threw this together to show how you could generate something totally custom and not related to "stages" or "step-load". I'll rework the wording here and maybe improve the maths.